octoprint_mrbeam/iobeam/laserhead_handler.py

```diff
@@ -342,7 +348,8 @@ def _write_laser_heads_file(self, file=None):
             laser_heads=self._lh_cache,
             correction_settings=self._correction_settings,
             last_used_lh_serial=self._current_used_lh_serial,
-            last_used_lh_model=self._current_used_lh_model,
+            last_used_lh_model_string=self._current_used_lh_model_string,
+            last_used_lh_model_id=self._current_used_lh_model_id,
```

Comment:
Please do not write 2 values which could potentially clash with each other. If `_current_used_lh_model_string` is a derivative of `_current_used_lh_model_id` then it should not be saved. At most 1 of the 2 should be saved, but not both.

Reply:
That is what I said in the standup and we talked about it and the result was to have both saved

Reply:
ah.. what was the reason already?